index.js

```diff
@@ -72,30 +73,35 @@ proto.read = function (readTree) {
   var self = this;
 
   return mapSeries(this.inputTrees, readTree)
-    .then(function(inputPaths) {
-      var inputTreeHashes = [];
-      var invalidateCache = false;
-      var keys, dir, updateCacheResult;
-
-      for (var i = 0, l = inputPaths.length; i < l; i++) {
-        dir = inputPaths[i];
-        keys = self.keysForTree(dir);
-        inputTreeHashes[i] = helpers.hashStrings(keys);
-
-        if (self._inputTreeCacheHash[i] !== inputTreeHashes[i]) {
-          invalidateCache = true;
-        }
-      }
+  .then(function(inputPaths) {
```

Comment:
Can you fix the indentation here?

Basically it should be something like the following:

```javascript
return mapSeries(this.inputTrees, readTree)
  .then(function(inputPaths) {
    /* stuff here */
  });
```